engine/src/main/java/org/hibernate/validator/internal/engine/constraintvalidation/ComposingConstraintTree.java

```diff
@@ -47,19 +47,19 @@
 	@Immutable
 	private final List<ConstraintTree<?>> children;
 
-	public ComposingConstraintTree(ConstraintDescriptorImpl<B> descriptor, Type validatedValueType) {
-		super( descriptor, validatedValueType );
+	public ComposingConstraintTree(ConstraintValidatorManager constraintValidatorManager, ConstraintDescriptorImpl<B> descriptor, Type validatedValueType) {
+		super( constraintValidatorManager, descriptor, validatedValueType );
 		this.children = descriptor.getComposingConstraintImpls().stream()
```

Comment:
I know that it was here already, but I'll still ask this - "to stream, or not to stream?" :smiley:

Reply:
Yeah, true that :).

The thing is this PR was already big so I refrained from doing any other changes.